src/aosm/azext_aosm/deploy/artifact.py

```diff
@@ -70,7 +73,7 @@ def _upload_to_storage_account(self, artifact_config: ArtifactConfig) -> None:
             with open(artifact_config.file_path, "rb") as artifact:
                 self.artifact_client.upload_blob(artifact, overwrite=True)
             logger.info(
-                f"Successfully uploaded {artifact_config.file_path} to {self.artifact_client.account_name}"
+                "Successfully uploaded %s to %s", artifact_config.file_path, self.artifact_client.account_name
```

Comment:
why did it not like the "f" string here and in a few other places? I am pretty sure "f" strings are the recommended way of making strings in Python

Reply:
Apparently not for the linting of this repo :(